README.md

```diff
@@ -47,6 +47,7 @@ In your `angular.json` the target `extract-i18n` that can be configured with the
 | `sourceLanguageTargetFile`   | Unused                                                      | If this is set (to one of the `targetFiles`), new translations in that target file will be set to `state="final"` (instead of default `state="new"`). This file can be used to manage changes to the source texts: when a translator updates the target, this tool will hint the developer to update the code occurrences.                                          |
 | `sourceFile`                 | `messages.xlf`. `ng add` tries to infer non default setups. | Filename (relative to `outputPath` of source translation file (e.g. `"translations-source.xlf"`).                                                                                                                                                                                                                                                                   |
 | `removeIdsWithPrefix`        | `[]`                                                        | List of prefix strings. All translation units with matching `id` attribute are removed. Useful for excluding duplicate library translations.                                                                                                                                                                                                                        |
+| `includeOnlyPrefix`          | `null`                                                      | Include only translations starting with given prefix. Useful for extracting translations of a single library in a multi-library project.                                                                                                                                                                                                                            |
```

Comment:
- Would it make sense to align the name with `excludeIdsWithPrefix` - eg `includeIdsWithPrefix`?
- Probably there are use cases where having this a list would be helpful?
- It should be documented that only one of the include/exlude options should be used?

Reply:
Good suggestions, thanks. Just pushed my changes to rename the property and made it a list. Also updated the README